src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,89 @@
+import { actions } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const [user, setUser] = useState<User>({
+    id: 0,
+    name: '',
+    email: '',
+    phone: '',
+  });
+
+  useEffect(() => {
+    // eslint-disable-next-line no-console
+    console.log(currentTodo);
+
+    if (currentTodo?.id) {
+      // eslint-disable-next-line no-console
+      console.log(currentTodo.id);
+
+      getUser(currentTodo.id).then((result: User) => setUser(result))
+        .catch(() => setUser({
+          id: 15,
+          name: 'User wasn\'t found',
+          email: 'User wasn\'t found',
+          phone: 'User wasn\'t found',
+        }));
+    }
+  }, [currentTodo]);
+
+  const dispatch = useDispatch();
+
+  const removeCurrentTodo = () => dispatch(actions.removeTodo());
+
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      <Loader />
+      {user?.name
+        ? (
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <div
+                className="modal-card-title has-text-weight-medium"
+                data-cy="modal-header"
+              >
+                {`Todo #${currentTodo?.id}`}
+              </div>
 
-      <div className="modal-card">
-        <header className="modal-card-head">
-          <div
-            className="modal-card-title has-text-weight-medium"
-            data-cy="modal-header"
-          >
-            Todo #3
-          </div>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                type="button"
+                className="delete"
+                data-cy="modal-close"
+                onClick={() => removeCurrentTodo()}
+              />
+            </header>
 
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <button type="button" className="delete" data-cy="modal-close" />
-        </header>
+            <div className="modal-card-body">
+              <p className="block" data-cy="modal-title">
+                {currentTodo?.title}
+              </p>
 
-        <div className="modal-card-body">
-          <p className="block" data-cy="modal-title">fugiat veniam minus</p>
+              <p className="block" data-cy="modal-user">
+                {/* For not completed */}
+                {currentTodo?.completed
+                  ? <strong className="has-text-success">Done</strong>
+                  : <strong className="has-text-danger">Planned</strong>}
 
-          <p className="block" data-cy="modal-user">
-            {/* For not completed */}
-            <strong className="has-text-danger">Planned</strong>
+                {' by '}
+
+                <a href={`mailto:${user.email}`}>
+                  {user?.name}
+                </a>
```

Comment:
Just don't render this part if you don't have a user

Reply:
Done